sites/public/src/ListingView.tsx

```diff
@@ -49,22 +49,9 @@ export const ListingView = (props: ListingProps) => {
     totalCount: t("t.totalCount"),
   }
 
-  const amiValues = listing?.unitsSummarized?.amiPercentages
-    ? listing.unitsSummarized.amiPercentages
-        .map((percent) => {
-          const percentInt = parseInt(percent, 10)
-          return percentInt
-        })
-        .sort(function (a, b) {
-          return a - b
-        })
-    : []
-
-  let groupedUnits: GroupedTableGroup[] = null
-
-  if (amiValues.length == 1) {
-    groupedUnits = getSummariesTable(listing.unitsSummarized.byUnitTypeAndRent)
-  } // else condition is handled inline below
+  const groupedUnits: GroupedTableGroup[] = getSummariesTable(
+    listing.unitsSummarized.byUnitTypeAndRent
```

Comment:
I thought the plan was to get the data from the unit summary table now?

Reply:
Yeah that is the plan - Patrick is working on the changes to `getSummariesTable` so that it can take a `UnitsSummary` (https://github.com/CityOfDetroit/bloom/pull/459), and then we'll be able to update this `ListingView` to pass in `listing.unitsSummary`.